src/main/java/com/thirdcc/webapp/service/impl/EventCrewServiceImpl.java

```diff
@@ -85,6 +122,38 @@ public EventCrewDTO save(EventCrewDTO eventCrewDTO) {
     @Override
     public void delete(Long id) {
         log.debug("Request to delete EventCrew : {}", id);
+        EventCrew eventCrew = eventCrewRepository
+            .findById(id)
+            .orElseThrow(() -> new BadRequestException("Cannot delete non existing event crew"));
+
         eventCrewRepository.deleteById(id);
     }
+
+    /**
+     * Assign user details(name and contact number) to the eventCrewDTO.
+     *
+     * @param eventCrewDTO to add on the user details' prop.
+     */
+    private EventCrewDTO mapUserDetails(EventCrewDTO eventCrewDTO) {
+        Optional<User> dbUser = userRepository.findById(eventCrewDTO.getUserId());
+        if(dbUser.isPresent()){
+            User user = dbUser.get();
+            eventCrewDTO.setUserName(user.getFirstName() + " " + (user.getLastName() != null? user.getLastName(): "") );
```

Comment:
if user dont have last name will become first name and empty space behind

Reply:
Resolved in dba827f85e1232353deb31e387ebb6ee1242cc45